openpathsampling/shooting.py

```diff
@@ -271,3 +271,11 @@ def probability_ratio(self, snapshot, old_trajectory,
         if new_snapshot is None:
             NEW_SNAPSHOT_SELECTOR.warn(stacklevel=3)
         return 1.0
+
+
+class VolumeSelector(ShootingPointSelector):
+    def __init__(self, volume):
+        self.volume = volume
+
+    def f(self, frame, trajectory):
+        return float(self.volume(frame))
```

Comment:
any particular reason you went for `float` instead of `int` casting?
(I understand it makes sense from a probablility point of view, but not really from the point of view that the output of volume should be `bool`)

Reply:
I'm thinking of it as a special case of a shooting point biasing function, like `BiasedSelector`. In that case you bias toward certain shooting points using a CV, which returns floats. Really, the actual value that comes out here doesn't matter, as long as it is either 0 or some constant $k$ (I took $k=1$).

FWIW, even `UniformSelector` technically returns `1.0` for its `f`. So this also keeps the typing consistent across different selectors.